src/backend/SynologyManager.ts

```diff
@@ -28,18 +28,34 @@ class SynologyManager {
         return [];
       }
 
-      if (config.synologyTagNames && config.synologyTagNames.length > 0) {
+      // Personal space album IDs (persons, concepts, geocoding) have priority over tags and regular albums
```

Comment:
Do these have to take precedence, or can they just all be included in the scraping?  Right now, the module is setup in such a way that, for tags, it pulls images with a tag from both the shared and personal spaces.

The end goal is simply a collection of photos to show, so I don't think we should preclude people from setting `synologyAlbumName` to grab all photos from an album, but also include personIds and conceptIds to pull images from those dynamically generated albums.

As I read this i realize that if you set both `synologyAlbumName` and `synologyTagNames`, the tags are preferred.  I added issue #13 to address this